### PR TITLE
Create resources and mock data for messages route

### LINKED ISSUE #35: As a user I want to be able to navigate to messages
- In ember we need a route to project/#/messages.
- In ember we need a messages page to display.

### PR DESCRIPTION
**Pull request by kscherling**
- Reorganize template and router folder structures to be hierarchically
correct
- Add message related templates and resources
- Stub basic message factories and binds
- Create fluid container component to appropriately pad content

Fixes #35

### CODE REVIEW

File: app/controllers/project/messages/message.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  model(params) {
+    return this.store.findRecord('message', params.message_id)
```
Comment (smashedlife) [inline app/controllers/project/messages/message.js]: This line is missing a semi-colon

File: app/mirage/config.js [hunk 2]
```diff
@@ -2,6 +2,26 @@ export default function() {
...
+  this.get('/messages', function(db /*, request */) {
+    return {
+      data: db.messages.map(attrs => (
+        {type: 'messages', id: attrs.id, attributes: attrs }
+      ))
+    };
+  });
+
+  this.get('/messages/:id', function(db, request) {
+    let id = request.params.id
```
Comment (smashedlife) [inline app/mirage/config.js]: This is missing a semi-colon

**Comment by smashedlife** [comment]
Just wondering why in some areas in the code, it is using var and in others let.

**Comment by kscherling** [comment]
Point made.  I will clean that up in another pr...

**Pull request closed without merging**